src/cache/QueryResultCache.ts

```diff
@@ -16,12 +16,14 @@ export interface QueryResultCache {
     disconnect(): Promise<void>
 
     /**
-     * Performs operations needs to be created during schema synchronization.
+     * Perform operations during schema synchronization.
      */
     synchronize(queryRunner?: QueryRunner): Promise<void>
 
     /**
-     * Caches given query result.
+     * Get data from cache.
+     * Returns cache result if found.
+     * Returns undefined if result is not cached.
```

Comment:
i think we can follow this

- https://jsdoc.app/tags-description
- https://jsdoc.app/tags-returns

Reply:
FYI, in TypeScript, we have [TSDoc](https://tsdoc.org/), which is supported by `tsc` and is very similar to JSDoc. Description does not exist, but there's a [@returns](https://tsdoc.org/pages/tags/returns/) tag indeed.

If need be, I can add `eslint-plugin-tsdoc` to the eslint config to prevent writing incorrect TSDoc.